package.json

```diff
@@ -24,6 +24,9 @@
     "@bazel/jasmine": "1.1.0",
     "@bazel/typescript": "1.1.0",
     "@types/jasmine": "3.5.0",
+    "@types/shelljs": "0.8.6",
+    "@types/mock-fs": "4.10.0",
+    "@bazel/bazel": "2.0.0",
```

Comment:
Did we decide to remove `@bazel/bazel` for now?

Reply:
ah yes, all removed now